package.json

```diff
@@ -8,7 +8,7 @@
     "seed": "node ./db/seeds/run-seed.js",
     "test": "jest",
     "prepare": "husky install",
-    "scratch": "psql -f scratch.sql > scratch.txt"
+    "scratch": "psql -f scratch/scratch.sql > scratch/scratch.txt"
```

Comment:
Was this an attempt to extract the SQL logic?

Reply:
I added the files to my .gitignore based on previous feedback, but for whatever reason .gitignore wouldn't actually ignore the files unless they were in a folder?